### ISSUE TITLE
spacemacs//cider-eval-in-repl-no-focus helper function unintended behavior

### ISSUE THREAD

**Issue by sooheon** [syl20bnr/spacemacs#3691]
This is a weird interaction, but it's 100% reproducible. 

With the clojure layer installed, on a fresh leiningen project, go to the core_test.clj file, change the test to pass, show the cider repl with `, s i`, and call `, t a` from core.clj. Every time, you will see the repl being polluted with integers which increment seemingly randomly:

<img width="732" alt="screen shot 2015-11-07 at 1 26 24 am" src="https://cloud.githubusercontent.com/assets/302945/11002177/92518bb8-84ee-11e5-9aeb-68ad54aed62f.png">

I tried to look into why this was happening, which led me to that helper function in the clojure layer. Weirdly, evaluating `(spacemacs//cider-eval-in-repl-no-focus (cider-ns-form))` with `M-:` prints similar numbers to the minibuffer.

AFAICT, all functions wrapped with `spacemacs//cider-eval-in-repl-no-focus` spam the minibuffer. `, s e`, for example, leaves `Indenting region... done` in the minibuffer (worthless information) while `C-c C-e` prints `=> #'blah.core/foo`, some feedback on sexp evaluation result. `, t a` is simply the most egregious one, spamming both the minibuffer and the cider-repl with those numbers. This can cause UX pain, because for tests that take a long time, `C-c ,` prints `Testing...` to the minibuffer until results appear, giving important feedback, while `, t a` just prints `Indenting region...done`.

Could the implementer of this wrapper function (git blame says its @syl20bnr :p) explain the need? All I can see is that it seems to distinguish between switching focus to repl and not, but vanilla cider commands like `cider-eval-last-sexp` don't switch focus by default, so all `spacemacs//cider-eval-in-repl-no-focus` can just be removed, no?

Basically I'm proposing getting rid of it and just using default cider function equivalent bound to those keys.
